microsite/data/plugins/analytics-module-ga.yaml

```
@@ -4,7 +4,7 @@ author: Spotify
 authorUrl: https://github.com/spotify
 category: Monitoring
 description: Track usage of your Backstage instance using Google Analytics.
```

```suggestion
description: Track usage of your Backstage instance using Google Analytics 4.
```
Should we mention that it uses GA 4 like this? Just to make it super clear.